tests/t_json

```diff
@@ -12,14 +12,14 @@ test_json() {
     PARAMS="$1"
     EXPECTED=$(echo "$2" | jq 'sort')
     RESULT=$($SDCV $PARAMS | jq 'sort')
```

Comment:
I think quoting $PARAMS here should solve any quoted path issues.

```bash
RESULT=$($SDCV "$PARAMS" | jq 'sort')
```

Reply:
No this variable contain several parameters, you can quote it as whole.

Reply:
...on the other side, this can be done better, see https://github.com/Dushistov/sdcv/pull/35